nixos/modules/system/boot/systemd.nix

```diff
@@ -477,6 +475,12 @@ in
       description = "Default unit started when the system boots.";
     };
 
+    systemd.ctrl-alt-del = mkOption {
```

Comment:
Happy to change that - let's figure out whether this option stays around at all. :)